components/camel-aws2-cw/src/main/docs/aws2-summary.adoc

```diff
@@ -0,0 +1,33 @@
+= AWS2 Component
+:docTitle: AWS 2
+:summary-group: AWS 2
+
+*Since Camel {since}*
```

Comment:
There's no `since` attribute, and I think the AWS2 components are being added gradually, so I don't think this line is appropriate.

Reply:
Ok, this is the same case for both AWS and Google summary as well then. I will make the change.